XWalkView/XWalkView/XWalkStubGenerator.swift

```diff
@@ -69,3 +69,41 @@ class XWalkStubGenerator {
         return "function(\(list)) {\n    \(body)\n}"
     }
 }
+
+private extension NSNumber {
+    var isBool: Bool {
+        get {
+            let trueNumber = NSNumber(bool: true)
+            let falseNumber = NSNumber(bool: false)
+            if (self.isEqualToNumber(trueNumber) && self.objCType == trueNumber.objCType)
+            || (self.isEqualToNumber(falseNumber) && self.objCType == falseNumber.objCType) {
+                return true
+            }
+            return false
+        }
+    }
+}
+
+private func toJSONLiteral(object: AnyObject, isPretty: Bool=false) -> String {
+    switch object {
+    case is NSNull:
+        return "null"
+    case is NSError:
+        return "\(object)"
+    case let number as NSNumber:
+        if number.isBool {
+            return (number as Bool).description
+        } else {
+            return (number as NSNumber).stringValue
+        }
+    case is NSString:
+        return object as String
```

Comment:
return "'\(object)'"
so we can remove the condition in XWalkStubGenerator (line 28 - 30)